week-4/implement/get-ordinal-number.test.js

```diff
@@ -2,18 +2,43 @@
 
 // continue testing and implementing getOrdinalNumber for additional cases
 // Write your tests using Jest - remember to run your tests often for continual feedback
-function getOrdinal(number){
-    const suffixes = ["th","st","nd","rd"];
-    const value = number % 100;
-    const sufix = (value > 10 && value <20) ? suffixes[0] : suffixes[Math.min(value%10,3)];
-    return value+sufix;
+function getOrdinal(number) {
+  if (typeof number !== 'number' || isNaN(number)) return null;
+
+  const suffixes = ['th', 'st', 'nd', 'rd'];
+  const value = number % 100;
+
+  return number + (suffixes[(value - 20) % 10] || suffixes[value] || suffixes[0]);
 }
-console.log(getOrdinal(1));   // "1st"
-console.log(getOrdinal(2));   // "2nd"
-console.log(getOrdinal(3));   // "3rd"
-console.log(getOrdinal(4));   // "4th"
-console.log(getOrdinal(11));  // "11th"
-console.log(getOrdinal(21));  // "21st"
-console.log(getOrdinal(42));  // "42nd"
-console.log(getOrdinal(103)); // "103rd"
-console.log(getOrdinal(112)); // "112th"
\ No newline at end of file
+describe('getOrdinal', () => {
+  test('returns "1st" for 1', () => {
```

Comment:
test.each would be very handy in here. 
Would be good not to miss numbers before 10 at least.

Reply:
Yes I try to learn test.each also if feels any difficulty I will let you know